chain/indexer/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 0.9.0
+
+* Introduce `IndexerShard` structure which contains corresponding chunks and `IndexerExecutionOutcomeWithReceipt`
+* `IndexerExecutionOutcomeWithReceipt` field `receipt` is always required
+* Introduce `IndexerExecutionOutcomeWithOptionalReceipt` which is the same as `IndexerExecutionOutcomeWithReceipt`
+but with optional `receipt` field.
```

Comment:
We expose it. Because users may want to use it while working with `IndexerTransactionWithOutcome`. `outcome` field there has type `IndexerExecutionOutcomeWithOptionalReceipt` so I would prefer to have this structure exposed.